README.md

```diff
@@ -1 +1,228 @@
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- A Code editor such as `VsCode`.
+- A web browser.
+- you need to have  `node` installed in your computer 
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git git@github.com:NurkaAmre/Final-Capstone-Front-End.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd Final-Capstone-Front-End
+  npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  npm start
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  rspec spec/
+```
+
+### Deployment <a name="triangular_flag_on_post-deployment"></a>
+
+You can deploy this project using: [Netlify](https://www.netlify.com/with/react/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Author <a name="authors"></a>
+
+<img src="https://ca.slack-edge.com/T47CT8XPG-U03REQGC0US-8675abab04f7-512" alt="Nurka" width="40" height="40" /> **Nurgul Kereikhan**
+
+- GitHub: [@githubhandle](https://github.com/NurkaAmre)
+- Twitter: [@twitterhandle](https://twitter.com/AmreNurgul)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/amre-nurgul/)
+
+<img src="https://ca.slack-edge.com/T47CT8XPG-U03NTSNV4RK-49df5661aad0-512" alt="Tanussri" width="40" height="40" /> **Tanusri Ghosh**
+
+- GitHub: [@chuaindia](https://github.com/chuaindia)
+- Twitter: [@chuaghosh](https://twitter.com/chuaghosh25)
+- LinkedIn: [@Ghosh](https://www.linkedin.com/in/tanusri-ghosh-2a56b814/)
+
+<img src="https://avatars.githubusercontent.com/u/105460631?v=4" alt="Joseph" width="40" height="40" /> **Joseph Irirwanirira**
+
+- GitHub: [@githubhandle](https://github.com/Irirwanirira)
+- Twitter: [@twitterhandle](https://twitter.com/iri_joseph)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/Joseph Irirwanirira)
+
+<img src="https://ca.slack-edge.com/T47CT8XPG-U03KHBLE332-7b0c9c134114-512" alt="Omar Mohammad Warraich" width="40" height="40" /> **Omar Mohammad Warraich**
+
+- GitHub: [@OmarMWarraich](https://github.com/OmarMWarraich)
+- Twitter: [@omarwarraich1](https://twitter.com/omarwarraich1)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/o-va)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Add Reservation time:** The reservation can be automatically cancel when time is over.
+- [ ] **User can visit the some Libary users:** .
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/NurkaAmre/Final-Capstone-Front-end/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
```

Comment:
- [ ] Since the last review, this pull request has not suffered any changes or commits.  Kindly implement the comments featured on the last review.  👍